test/system/preTestScript.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+
```

Comment:
I think you want a `set -e` here

Reply:
using sed -i , when needed.

Reply:
I cannot use it here, though , we have a command `jarvis save $CLUSTER_NAME`. It gives the following log, even though the execution is susccessful. 
```
jarvis save glorian
[jarvis login] failed: Post https://10.240.124.131/auth/v1/login: dial tcp 10.240.124.131:443: getsockopt: connection refused
Post https://10.240.124.131/auth/v1/login: dial tcp 10.240.124.131:443: getsockopt: connection refused
```
If i add ` set -e` , the execution stops and fails here, which is not necessary, as there is no problem with the function of the command.